pkl-core/src/main/java/org/pkl/core/parser/ast/StringPart.java

```diff
@@ -31,15 +32,42 @@ public StringPart(Span span, @Nullable List<? extends @Nullable Node> children)
     return visitor.visitStringPart(this);
   }
 
-  public static final class StringConstantParts extends StringPart {
-    public StringConstantParts(List<StringConstantPart> parts, Span span) {
-      super(span, parts);
+  public static final class StringConstantPart extends StringPart {
```

Comment:
Maybe call this `StringChars`? `StringConstantPart` sounds too similar to the `StringConstant` node.